lib/console.js

```diff
@@ -134,6 +134,9 @@ Console.prototype.log = function log(...args) {
 };
 
 
+Console.prototype.debug = Console.prototype.log;
```

Comment:
Per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Console), 

> Starting with Chromium 58 this method only appears in Chromium browser consoles when level "Verbose" is selected.

Thus for consistency, I don't think we should show debug logs by default, per default. Instead we could perhaps only display debug logs when an environment variable is specified. @nodejs/collaborators thoughts?

Reply:
I like having debug be conditional upon some flag. Having it just be an alias doesn't seem that appealing.

Reply:
Firefox and I believe every other major browser do not require a flag for `console.debug()` to display.  I'm not sure we should follow Chrome's practice on this one.

Reply:
I think that choosing t to follow Chrome on this feature would mean getting a bit too much opinionated, and I don't think we should. As you said, no other browser makes this choice.
The choice to restrain visibility of the debug should be made by the users IMHO, and Node should only provide the method.

Reply:
I’m good with this. If they want to, anybody can override this method in their App, globally, and possibly depending on environment variables…